ansible/playbooks/adhoc/grow_root_vg/grow_root_vg.yml

```diff
@@ -163,3 +163,6 @@
 
   - name: Resize /var
     command: "xfs_growfs /var "
+
+  - name: Trigger filesystem metrics
+    shell: "docker exec -ti oso-rhel7-host-monitoring /usr/bin/cron-send-filesystem-metrics -v"
```

Comment:
This command runs with an additional flag on compute nodes so that we don't alert on customer PVs that are full:
`docker exec -i oso-rhel7-host-monitoring cron-send-filesystem-metrics --filter-pod-pv` . This should probably which group the node its running on is a member of and either run one or the other.

You could probably use a when clause for this like:
`when: "'oo_subhosttype_compute' in group_names"` for compute and `when: "('oo_subhosttype_infra' in group_names) or ('oo_hosttype_master' in group_names)"` for masters and infras.

Reply:
Can the `--filter-pod-pv` always be set or does it specifically only get set for compute?

Reply:
It shouldn't be set for infra nodes because this alerts us to when infra PVs are full (like elasticsearch and cassandra volumes). I think it matters a little less for the masters nodes (since they don't have any PVs anyway),